src/widgets/statistical-context-criteria/components/statistical-context-criteria.jsx

```diff
@@ -16,9 +16,118 @@ const {
   HORIZONTAL_CLASS,
 } = WIDGET_STATISTICAL_CONTEXT_CRITERIA;
 
+const StatisticalContextCriteria = props => {
+
+  const {
+    formName,
+    path,
+    selectedSerie,
+    selectedOperation,
+    campaigns,
+    operations,
+    series,
+    multipleCampaign,
+    required,
+    focusOnInit,
+    horizontal,
+  } = props;
+
+  const [selectedSerieState, setSelectedSerieState] = useState(selectedSerie);
+  const [selectedOperationState, setSelectedOperationState] = useState(selectedOperation);
+
+	useEffect(() => {
+    props.loadSeriesIfNeeded();
+    if (operations &&  selectedSerie)
+     props.loadOperationsIfNeeded(selectedSerie);
+    if (campaigns && selectedOperation)
+     props.loadCampaignsIfNeeded(selectedOperation);
+    if (campaigns && selectedOperation)
+     props.loadCampaignsIfNeeded(selectedOperation);
+  }, []);
```

Comment:
Are you sure this dependencies are correct ?
You should install the eslint plugin dedicated to hooks it will help you.